libsel4/tools/condition.py

```diff
@@ -0,0 +1 @@
+../../tools/condition.py
```

Comment:
What is this, so we really need this indirection and the ".."? Don't we have the search path somewhere?

Reply:
This is making a symlink to condition.py, I'm not sure why it shows the path as its contents.
I just went with how it seemed to be done in libsel4/tools, where the rest of these files are all symlinks.

Reply:
Thanks for the explanation. If this just follows existing stuff, then it's fine. And symlink within the same repo will always work.

Improving the python usage would be a different PR indeed. You could create an issue to track this ;)